src/content/general/engine/scripting/readme.md

```diff
@@ -177,6 +177,63 @@ Several control structures implicitly return the value of their final expression
 )
 ```
 
+## Object Type Masks
+Several functions from Halo 2 onwards allow the user to perform actions on all objects of a certain type (e.g. bipeds, vehicles etc) that are currently loaded in a scenario. These functions use long values to determine which object types should be referenced.
+
+| Type Mask <long>| Object Type     |
+| ----------------| --------------- |
+| 1               | biped           |
+| 2               | vehicle         |
+| 4               | weapon          |
+| 8               | equipment       |
+| 1024            | crate           |
+
+These values can be added together to effect multiple object types at once. For example, using the function ```(object_destroy_type_mask 5)``` would destroy all bipeds and weapons loaded on the map. Both 0 and 1039 will destroy all objects of the above listed types. The functions which support this are:
+
+```hsc
+(<object_list> volume_return_objects_by_type <trigger_volume> <type_mask>) ; Halo 2 onwards
+(<void> object_destroy_type_mask <type_mask>) ; Halo 2 onwards
+(<void> add_recycling_volume_by_type <trigger_volume> <long> <long> <type_mask>) ; Halo Reach onwards
+(<void> add_offscreen_recycling_volume_by_type <trigger_volume> <long> <long> <type_mask>) ; Halo 4 onwards
+```
+Below is a full listing of object type mask combinations: 
+<!--- this can be stripped if it's a bit too long. --->
+
+| Type Mask <long>| Object Type                             |
```

Comment:
`<long>` won't render properly because it gets interpreted as an HTML tag. You can change this to `(long)` to avoid this, or use escaped codes: `&#60;long&#62;`